integration_tests/jacoco-offline/build.gradle

```diff
@@ -0,0 +1,112 @@
+import org.robolectric.gradle.AndroidProjectConfigPlugin
+
+apply plugin: 'com.android.library'
+apply plugin: AndroidProjectConfigPlugin
+apply plugin: "jacoco"
+
+android {
+    compileSdk 33
+
+    defaultConfig {
+        minSdk 16
+        targetSdk 33
+    }
+
+    compileOptions {
+        sourceCompatibility = '1.8'
+        targetCompatibility = '1.8'
+    }
+
+    testOptions.unitTests.includeAndroidResources true
```

Comment:
Should you add `no-verify` like https://github.com/robolectric/robolectric/issues/8001#issuecomment-1455118413 here?

Reply:
I can't determine it when I don't know whether Robolectric will do harm to class bytecode.
IMO, it would be better to integration test with JVM bytecode verification so that any bytecode conflicts or problems can be found early. 
And `no-verify` should be used to debug and trace down to get a nicer stacktrace if needed.

Reply:
Yeah we shouldn't need to use no-verify.

Reply:
Okay, I got it.